### ISSUE TITLE
modern systemd enumerators and services per driver

### ISSUE THREAD

**Issue by pavlozt** [ntd/ansible-role-nut#14]
In modern versions of systemd it is necessary correctly process service names

## Error

for example, if Idefine single UPS named "apc"
```
    nut_ups:
      - name: apc
         ...
```
got error :

```
RUNNING HANDLER [ntd.nut : restart nut] **********************************************************************************************************************************
failed: (item=nut-driver) => {"ansible_loop_var": "item", "changed": false, "item": "nut-driver", "msg": "Could not find the requested service nut-driver: host"}
```

because there are no "nut-driver" service in debian 12, but created named "nut-driver@apc" service.

## Workaround
Define additional variable :
```
    nut_services:        # Name of the services to enable
      - nut-driver@apc
      - nut-monitor
      - nut-server
```

**Comment by pavlozt** [comment]
Ansible does not yet support globs in the systemd module.
It is unknown when support for the numerator system will appear, so I suggest slightly modifying examples in the Readme for now:

```
nut_services:
  - nut-driver@apc
  - nut-monitor
  - nut-server

```

**Comment by rpavlik** [comment]
I managed to handle this in #15 though it very well may be not-idiomatic.

**Comment by ntd** [comment]
Fixed in #15.
